pkgs/applications/office/timeline/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, fetchurl
+, python3Packages
+, makeDesktopItem
+, copyDesktopItems
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "timeline";
+  version = "2.4.0";
+
+  src = fetchurl {
+    url = "mirror://sourceforge/thetimelineproj/${pname}-${version}.zip";
+    sha256 = "1pa0whqci6c2p20xf7gbndrrpr1xg42ixhql595ibdd4p3l37v23";
+  };
+
+  nativeBuildInputs = [ python3Packages.wrapPython copyDesktopItems ];
+
+  pythonPath = with python3Packages; [
+    wxPython_4_0
+    humblewx
+    icalendar
+    markdown
+    pysvg-py3
+    pillow
+  ];
+
+  desktopItems = [
+    (makeDesktopItem {
+      desktopName = "Timeline";
+      name = "timeline";
+      comment = "Display and navigate information on a timeline";
+      icon = "timeline";
+      terminal = false;
+      exec = "timeline";
+      categories = "Office;Calendar;";
+    })
+  ];
+
+  dontBuild = true;
+  doCheck = false;
```

Comment:
Please add a comment why you disabled tests.

Reply:
Oh, i had the tests running with python 2, which lead to strange errors like

`ImportError: No module named builtins`

Now 159 tests fail because they need an X server.

```
running install tests
...........................E...E..............E...E...........E.....E.....E...E....E..............E...E...E.....E.........E.E.E..........E.....E..........E.......E.....................E..E.E............E.................E..E.....E....E..E...............................E....EE........E....EE...E......EE..............EE....E....................E...................E.......E.....E..................................E......................EE................E.............................E.E..E.......E.......EE..........E.....EE....E....E.....E......EE..........E...E.E.......E..........E...................E.........E.......E.EE................E.......E.......E....E...E.....EE........................E.E.E.....E..............EE....E..........E...E.E..E..........E................E.......E......E.E....E.................E..E.E..E...E.....EE.....E..E.........E....E..................E....E........................E.....................................E................EE......EE.......................................................................E.........................E......E..............E...E........E.E.E..........E.......E.....E........E................E.E.........E..............................E..........E..........E.........E...E................E..E............EE..E...........E...E.................E......E........E......E......E........E...E.................E.....E.E......E....................E..E.EE......E........
======================================================================

...
ERROR: test_can_uncheck_show_time (unit.wxgui.dialogs.changenowdate.controller.describe_change_now_date_dialog_controller)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/build/timeline-2.4.0/test/unit/wxgui/dialogs/changenowdate/controller.py", line 31, in setUp
    WxAppTestCase.setUp(self)
  File "/build/timeline-2.4.0/source/timelinelib/test/cases/wxapp.py", line 27, in setUp
    self.app = self.get_wxapp()
  File "/build/timeline-2.4.0/source/timelinelib/test/cases/unit.py", line 97, in get_wxapp
    return wx.App(False)
  File "/nix/store/aqfnca9v3r1ax6nar45q91zfm60iibyz-python3.8-wxPython-4.0.7.post2/lib/python3.8/site-packages/wx/core.py", line 2102, in __init__
    raise SystemExit(msg)
SystemExit: Unable to access the X Display, is $DISPLAY set properly?

----------------------------------------------------------------------
Ran 1431 tests in 4.814s

FAILED (errors=159)
```
not sure if nixos module tests are worth it. until now nobody cared about this program. i think it's quiet mature and popular, so maybe